documentation/markdown/usage/account/json-api.md

```diff
@@ -0,0 +1,281 @@
+# Account management JSON API
+
+Everything related to account management is done through a JSON API,
+of which we will describe all paths below.
+There are also HTML pages available to handle account management
+that use these APIs internally.
+Links to these can be found in the HTML controls
+All APIs expect JSON as input, and will return JSON objects as output.
+
+## Finding API URLs
+
+All URLs below are relative to the index account API URL, which by default is `http://localhost:3000/.account/`.
+Every response of an API request will contain a `controls` object,
+containing all the URLs of the other API endpoints.
+It is generally advised to make use of these controls instead of hardcoding the URLs.
+Only the initial index URL needs to be known then to find the controls.
+Certain controls will be missing if those features are disabled in the configuration.
+
+## API requests
+
+Many APIs require a POST request to perform an action.
+When doing a GET request on these APIs they will return an object describing what input is expected for the POST.
+
+## Authorization
+
+After logging in, the API will return a `set-cookie` header.
+This cookie is necessary to have access to many of the APIs.
+When including this cookie, the controls object will also be extended with new URLs that are now accessible.
+When logging in, the response body JSON body will also contain a `cookie` field containing the cookie value.
```

Comment:
The structure of that field should still be described (`css-account=xxx`).

Reply:
Also perhaps, in the body (where I got the field value from), we shouldn't call the field `cookie` but `authorization` or `secret` or similar. Or maybe just not have it at all, and move auth entirely to the headers.